src/components/Header/GlobalNav.js

```diff
@@ -19,25 +18,23 @@ export const GlobalNav = ({ light = false, toggle = false, setToggle }) => {
   const router = useRouter()
   const { asPath, pathname, push } = router
   const { viewport, setResponsive } = useResponsive()
-  // TEMPORARY
-  const { dataset, getDataset } = useDatasetManager()
+  const { dataset, datasetId, getDataset, getTotalSamples } =
+    useDatasetManager()
```

Comment:
Did the formatter/linter do this awkward linebreak?

Reply:
Yes.

Reply:
This formatting is used (included the code merged into `dev`). Please let me know if it need to be re-configured and if so, let's reformat the files in a new issue.

Reply:
No I think that is fine, just stuck out and was curious.